app/test_main.py

```diff
@@ -1,3 +1,42 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_zero_age_for_both() -> None:
+    result = get_human_age(0, 0)
+    assert result == [0, 0], f"Expected [0, 0], but got {result}"
+
+def test_age_below_15_for_both() -> None:
+    result = get_human_age(14, 14)
+    assert result == [0, 0], f"Expected [0, 0], but got {result}"
+
+def test_age_of_15_for_both() -> None:
+    result = get_human_age(15, 15)
+    assert result == [1, 1], f"Expected [1, 1], but got {result}"
+
+def test_age_of_23_for_both() -> None:
+    result = get_human_age(23, 23)
+    assert result == [1, 1], f"Expected [1, 1], but got {result}"
```

Comment:
This test case seems to repeat the scenario tested in 'test_age_of_15_for_both'. According to the checklist, make sure that your tests don't test the same case multiple times.